detox/src/utils/retry.js

```diff
@@ -1,5 +1,6 @@
 const sleep = require('./sleep');
 
+const DEFAULT_UNREF = false;
```

Comment:
you already have a default value for this method in its signature.. why to define it in another place? 
let's have a single source of truth

Reply:
In a second thought, it has some sense to separate between the configurations (I mean that `DEFAULT_UNREF ` is the specific default value of `retry.js`), but since you already have in tests the check that we don't use `unref`, it isn't necessary to explicitly define it in the implementation.

Reply:
Passing now `undefined`.